app/features-json/login_json_controller.rb

```diff
@@ -0,0 +1,48 @@
+require_relative "../shared/controller_base"
+require_relative "../services/user_service"
+require_relative "../services/dot_keys_variable_service"
+
+require "jwt"
+require "json"
+
+module FastlaneCI
+  # Controller responsible of handling the login process using JWT token.
+  class LoginJSONController < ControllerBase
+    HOME = "/login"
+
+    post HOME.to_s do
+      # Allow this endpoint to be requested with { Content-Type: application/json }
+      payload = params
+      payload = JSON.parse(request.body.read).symbolize_keys unless params[:path]
+      email = payload[:email]
+      password = payload[:password]
+      user = Services.user_service.login(email: email, password: password)
```

Comment:
let's make this `username` instead of `email`

Reply:
Early on we decided that usernames will always be email to reduce complexity. I don't know if it's a good idea to change that. I'd like to hear more thoughts on this.

Reply:
I don't particularly have a strong opinion on this. I just know that username gives us more flexibility. If we want to stick with emails that cool with me 👍

Reply:
Cool, let's stick with email. We can open it up in the future to usernames. Part of our onboarding involves grabbing the GitHub account info and creating a user from their email. We don't default to an empty box.